### ISSUE TITLE
Support the call of Ukraininan government and change “Russia” to “Russia is a terrorist state”

### ISSUE THREAD

**Issue by alanmels** [backdrop/backdrop-issues#5712]
## Description of the need

Here is the quote from the tweet of the @FedorovMykhailo, a Vice Prime Minister of Ukraine and Minister of Digital Transformation, https://twitter.com/FedorovMykhailo/status/1556523604099170304 

> Call to action to Ukrainian and world IT-community: support a flashmob [#RussiaIsATerroristState](https://twitter.com/hashtag/RussiaIsATerroristState?src=hashtag_click). Goal  — to change “Russia” for “Russia is a terrorist state” in all possible web portal & online services, where you need to choose language, delivery etc. The first results below.

## Proposed solution

Most of us in the West live our daily lives in peace and prosperity and even-though might have a huge sympathy and empathy towards ordinary Ukrainians going through horrible war instigated by a terrorist mafia state of Russia, do not know how to help, do not see what we as a Backdrop community could do to help Ukraine. Well, here is our chance - we could change all occurrences of `Russia` in the core to `Russia is a terrorist state` and believe me that would be huge moral support to Ukraine. 

And if it's picked up by news outlets, then it could even turn into something that would make Backdrop even popular. At least, I personally would use all my connections in Ukraine (and I do have them, I traveled through the country, have lot's of friends, in fact one of former AltaGrade member is fighting the war to protect his country) to start using Backdrop in Ukraine more actively.

I understand that if the request comes as too radical and won't be accepted, but I thought that it's still worth a try.

**Comment by BWPanda** [comment]
I understand the desire to support Ukraine (and, more generally, those being persecuted, silenced, etc.), but I fail to see how changing a text string in Backdrop core will "be huge moral support to Ukraine". How will that text string affect real men and women overseas who are having real problems?

The best I can see it doing is to raise awareness of the Ukranian situation. But if all the current campaigns around the world trying to do the same thing aren't working, how will a Backdrop string help?

Yes I'm pessimistic, but I don't think Backdrop core should get involved in politics, etc. So I vote no.

This could very easily be a contrib module though...

**Comment by kiamlaluno** [comment]
That change won't be visible until a new release is created with that change and that release is used in sites. The change would then be only visible to users that select a country in a Backdrop site, probably, since Backdrop doesn't show a country name in many places. (Even in that case, the change would be shown only when the country name is _Russia_.)

I would let the site owners decide about the messages they want to give to their users. I personally find not so nice to show, for example, to users who are selecting in their user profile their country is Russia, *Russia is a terrorist state*.

**Comment by klonos** [comment]
I too find this very radical and not suitable for core. If we were to do that for every country that in the past has invaded other countries, then we'd need to add similar text for many other country instances - I just don't see how that would provide any real help to those that have suffered and/or still suffering 🤷🏼

I understand the desire to push for resolution in the Ukraine/Russia war, but at the end of the day, calling terrorists "terrorists" historically never stopped them from doing what they have been doing. I would say leave all that to politics/politicians.

**Comment by stpaultim** [comment]
There have been a number of situations in which the Backdrop CMS community has been asked to make a statement about a political or human rights issue. This is always difficult because our community is quite diverse, geographically and idealogically. 

As a member of the Project Management Committee, I'm reluctant to try and speak for the entire project on issues that are not specific to our project. However, I think that there are alternative ways for our community to voice our values. I struggle to decide when to do this and the best means. 

I'd be interested in "contrib" based actions. I noticed that a "Ukraine" project has been started (by @alanmels) in the contrib space. 
https://github.com/backdrop-contrib/ukraine/blob/main/LICENSE.txt

I would be interested in collaborating with @alanmels or others to come up with some way to use this "potential" module to either take action or make a statement. As a contrib project, participation would be voluntary. I would also be willing to participate in a voluntary community based statement that individuals could sign on to, or a petition. A voluntary fundraiser might also be possible. 

I'm inclined to take discussion of this to the issue queue for the contrib module for now and discuss possible statements or actions there.

In short, I'm not yet ready to sign-on to this specific proposal. But, I am willing to discuss alternative actions. Feel free to reach out to me publicly or privately for ongoing discussions.

**Comment by stpaultim** [comment]
I make a comment in the Ukraine issue queue:
https://github.com/backdrop-contrib/ukraine/issues/1

**Comment by alanmels** [comment]
Thank you all for your inputs. I quickly posted on https://github.com/backdrop-ops/backdrop-community/issues/24. Unfortunately, I really do not have time, but will try to come up with the module soon. Hopefully.
